silver/models/payment_methods.py

```diff
@@ -60,6 +62,11 @@ def as_list(cls):
 
     objects = InheritanceManager()
 
+    @property
+    def one_way_fields(self):
```

Comment:
I know, but that's what they are. They're not final, otherwise you shouldn't be able to change them at all. And definitely not locked or imutable.